gateway/src/cluster/builder.rs

```diff
@@ -184,6 +185,22 @@ impl ClusterBuilder {
         self
     }
 
+    /// Set specific shard presences to use when identifying with the gateway.
+    ///
+    /// Accepts a closure. The closure accepts a [`u64`] and returns an
+    /// [`Option<UpdatePresencePayload>`]. This presence will override any set
+    /// by [`presence`], even if the function returns [`None`].
```

Comment:
Nit for consistency:
```suggestion
    /// by [`presence`], even if the provided closure returns [`None`].
```